packages/boxel-ui/addon/src/components/select/usage.gts

```diff
@@ -46,7 +46,8 @@ export default class BoxelSelectUsage extends Component {
 
   <template>
     {{! template-lint-disable require-scoped-style }}
-    <style>
+    <style
+    >
```

Comment:
@tintinthong is this intentional?

Reply:
Its prettier thing. But note I put 

```
    {{! template-lint-disable require-scoped-style }}
```

on style tags bcos `pnpm lint:fix` adds the scoped attribute on some style tags which are meant to be global. But this is definitely not the intention particularly for dropdown elements which gets wormholed to the top of the dom. So the disabling of template lint is very much intentional.